src/material-experimental/mdc-list/list-styling.ts

```diff
@@ -6,15 +6,39 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive} from '@angular/core';
+import {Directive, Inject, Optional} from '@angular/core';
+import {LIST_OPTION, ListOption} from './list-option-types';
+
+/**
+ * Directive whose purpose is to add the MDC list-item `graphic` or `meta`
+ * class depending on checkbox position in list options.
```

Comment:
Could this comment be expanded to elaborate more on the context here? (i.e. why this is necessary)